src/transformers/models/whisper/generation_whisper.py

```diff
@@ -652,10 +606,17 @@ def generate(
                 batch_idx_map=batch_idx_map,
             )
 
-            # 6.5 prepare decoder input ids
-            suppress_tokens = _get_attr_from_logit_processors(
-                logits_processor, SuppressTokensLogitsProcessor, "suppress_tokens"
-            )
+            # 6.3 prepare decoder input ids
+            if logits_processor is not None: 
```

Comment:
Is this `if/else` logic required? It should be handled by `_get_attr_from_logig_processors` right?
https://github.com/huggingface/transformers/blob/38aa42d459ad5a904077ecf683fb5b670f4f2540/src/transformers/models/whisper/generation_whisper.py#L121-L123